components/HeroSection.vue

```diff
@@ -18,19 +18,19 @@
         >
           <v-heading
```

Comment:
Reduce the min height on the section so that the section gradient below is visible.

https://user-images.githubusercontent.com/11095038/169996600-7a2ec96a-040c-46fc-b9cb-615bf308e609.mov